dev-java/netty-common/netty-common-4.0.36-r1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+MY_PN="netty"
+MY_P="${MY_PN}-${PV}"
+DESCRIPTION="Async event-driven framework for high performance network applications"
+HOMEPAGE="https://netty.io/"
+SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/${MY_P}.Final.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="dev-java/commons-logging:0
+	dev-java/javassist:3
+	dev-java/log4j:0
+	dev-java/slf4j-api:0"
+
+RDEPEND="virtual/jre:1.8
+	${CDEPEND}"
+
+# It does not build with java 11
```

Comment:
please add info what is the cause of the failures - missing classes, incompatibility, ...

Reply:
Done